CoffeeChat/Controller/CommunityViewController.swift

```diff
@@ -10,23 +10,142 @@ import UIKit
+            firstName: "Manish",
+            googleID: "12345",
+            graduationYear: "2021",
+            hometown: "Boston, MA",
+            interests: ["Art", "ABC", "Chill", "Box", "Long", "Array"],
+            lastName: "Shah",
+            major: "Computer Science",
+            matches: nil,
+            netID: "llx2",
+            profilePictureURL: "",
+            pronouns: "He/Him",
+            facebook: nil,
+            instagram: nil
+        )
+    ]
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        view.backgroundColor = .backgroundLightGrayGreen
+        view.backgroundColor = .backgroundLightGreen
+
+        searchBar.delegate = self
+        searchBar.backgroundColor = .backgroundWhite
+        searchBar.backgroundImage = UIImage()
+        searchBar.placeholder = "Search"
+        searchBar.searchTextField.backgroundColor = .backgroundWhite
+        searchBar.searchTextField.textColor = .black
+        searchBar.searchTextField.font = ._16CircularStdBook
+        searchBar.searchTextField.clearButtonMode = .never
+        searchBar.layer.cornerRadius = 8
+        searchBar.showsCancelButton = false
+        searchBar.layer.shadowColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.15).cgColor
+        searchBar.layer.shadowOpacity = 1
+        searchBar.layer.shadowRadius = 4
+        searchBar.layer.shadowOffset = CGSize(width: 0, height: 2)
+        view.addSubview(searchBar)
 
-        // Do any additional setup after loading the view.
+        communityTableView.delegate = self
+        communityTableView.dataSource = self
+        communityTableView.isScrollEnabled = true
+        communityTableView.separatorStyle = .none
+        communityTableView.backgroundColor = .clear
+        communityTableView.rowHeight = UITableView.automaticDimension
+        communityTableView.estimatedRowHeight = 140
+        communityTableView.sizeToFit()
+        communityTableView.register(CommunityUserTableViewCell.self, forCellReuseIdentifier: CommunityUserTableViewCell.reuseIdentifier)
+        view.addSubview(communityTableView)
+
+        setupConstraints()
     }
-    
 
-    /*
-    // MARK: - Navigation
+    private func setupConstraints() {
+
+        searchBar.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(23)
+            make.leading.trailing.equalTo(communityTableView)
+            make.height.equalTo(40)
+        }
 
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+        communityTableView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(24)
+            make.top.equalTo(searchBar.snp.bottom).offset(20)
+            make.bottom.equalToSuperview()
+        }
```

Comment:
little thing, but it feels a kind of strange to me to have the `leading/trailing` of the search bar rely on the table view (and also the fact that making the constraints for the tableview appears after the constraints for the search bar), and then have the `top` of the table view rely on the search bar.